client/packages/system/src/Item/api/operations.graphql

```diff
@@ -47,6 +47,13 @@ fragment StockLine on StockLineNode {
     id
     dosesPerUnit
   }
+  vvmStatus {
+    __typename
+    id
+    level
+    unusable
+    description
+  }
```

Comment:
This node needs to be added also because mapping in useHistoricalStockLines uses this api to remap draft prescription lines